pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -78,6 +78,9 @@ def work(self):
                             return False
 
                         items_stock = self.bot.current_inventory()
+                        berry_id = 701 # @ TODO: use better berries if possible
```

Comment:
I don't know how git hub PR works.

But it automatically added my second PR to this one. lol

On Fri, Jul 29, 2016 at 1:27 AM, Eli White <notifications@github.com> wrote:

> In pokemongo_bot/cell_workers/pokemon_catch_worker.py
> <https://github.com/PokemonGoF/PokemonGo-Bot/pull/1536#discussion_r72743050>
> :
>
> > @@ -78,6 +78,9 @@ def work(self):
> >                              return False
> >
> >                          items_stock = self.bot.current_inventory()
> > +                        berry_id = 701 # @ TODO: use better berries if possible
>
> This change seems irrelevant to this PR.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PokemonGoF/PokemonGo-Bot/pull/1536/files/c7ae1af24d13efb6b221cb9be3b173d72abb7bed#r72743050>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATetbjyM1Kg7Pd7mySSLRS-zip1WfBpzks5qaY8-gaJpZM4JX1eg>
> .
>